lambda-function/src/utils/resultutils.ts

```diff
@@ -0,0 +1,19 @@
+interface ErrorMessage {
+  timestamp: Date;
+  status: string;
+  message: string | undefined;
+  validationErrors: any;
+}
+
+export const parseError = (message: string): string => {
+  try {
+    const err: ErrorMessage = JSON.parse(message);
+    if (err.message) {
+      console.log("########## undefined", err.message);
```

Comment:
What does 'undefined' mean here?

Reply:
Oops.. left some debuggign code.. also added a comment to make it more clear